wiki/Tournaments/MWC/2014/en.md

```diff
@@ -3,136 +3,131 @@ osu!mania World Cup 2014
-| Tournament Management | ![][flag_DE] [Loctav](https://osu.ppy.sh/u/71366) // ![][flag_DE] [p3n](https://osu.ppy.sh/u/123703) // ![][flag_ES] [Deif](https://osu.ppy.sh/u/318565)                                                                                                                                                  |
-| Map Selectors         | ![][flag_RU] [\[Shi-Ra\]](https://osu.ppy.sh/u/2008489) // ![][flag_CL] [Agka](https://osu.ppy.sh/u/64811)                                                                                                                                                                                                            |
-| Streamers             | ![][flag_AU] [peppy](https://osu.ppy.sh/u/2) // ![][flag_PL] [Marcin](https://osu.ppy.sh/u/722665) // ![][flag_FR] [shARPII](https://osu.ppy.sh/u/776257)                                                                                                                                                 |
-| Commentators          | ![][flag_US] [Charles445](https://osu.ppy.sh/u/85000) // ![][flag_FR] [Mr Color](https://osu.ppy.sh/u/116078) // ![][flag_IT] [Roxas](https://osu.ppy.sh/u/1041833) // ![][flag_US] [ztrot](https://osu.ppy.sh/u/6347) // ![][flag_RU] [\[Shi-Ra\]](https://osu.ppy.sh/u/2008489) |
-| Statistician          | ![][flag_PL] [Marcin](https://osu.ppy.sh/u/722665)                                                                                                                                                                                                                                                                                |
+In every world cup, people conquering a place on the podium are eligible to receive unique prizes created specifically for World Cup winners. It is possible that these items change with every installment of the World Cups.
 
-------------------------------------------------------------------------
+| Placing | Prize(s) |
+|:---:|---|
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place")   | 6 month supporter tag, profile badge, osu! merchandise, "osu!mania Champion" usertitle |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 3 month supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month supporter tag |
+
+## Organization
 
-Links
------------
+The osu!mania World Cup 2014 is run by various community members by distributing the multitude of tasks into various fields of responsibility.
 
--   [osu!mania World Cup 2014 on Twitch](https://www.twitch.tv/osulive/)
--   [Registration Form](https://docs.google.com/forms/d/1mNcRLbrD7dN9ZbToFiBdFqPcdW4_T8aotASGPOXJjZY)
--   [Discussion Thread](https://osu.ppy.sh/forum/t/222100)
--   [Statistics: 4K Bracket](https://mwc.servegame.com/results/view/1)
--   [Statistics: 7K Bracket](https://mwc.servegame.com/results/view/2)
+| Job | Person(s) |
```

Comment:
i would use `| Position | Member(s) |` as in the OWC2017 article 

*also Persons 🤔*